### DISCUSSION TITLE
Assenze che non rientrano nel calcolo.

### DISCUSSION THREAD

**Discussion by manuelbarberi** [iisgiua/giuaschool#260] [Idee e proposte]
Salve, quando inserisco un'assenza, sarebbe possibile detrarla dal calcolo generale?

**Comment by trinko** [comment]
No, non è possibile.

Unico modo per aggirare il problema è: togliere l'assenza e inserire una ANNOTAZIONE che indica l'assenza. In questo modo, l'assenza è inserita formalmente sul registro, ma non viene considerata nè per i calcoli, né per le giustificazioni.
